sdk/storage/Azure.Storage.Files.DataLake/api/Azure.Storage.Files.DataLake.netstandard2.0.cs

```diff
@@ -559,6 +559,7 @@ public partial class DataLakeFileAppendOptions
         public bool? Flush { get { throw null; } set { } }
         public string LeaseId { get { throw null; } set { } }
         public System.IProgress<long> ProgressHandler { get { throw null; } set { } }
+        public Azure.Storage.UploadTransferValidationOptions TransferValidationOptions { get { throw null; } set { } }
```

Comment:
This doesn't look to be in the correct place.

Reply:
@jaschrep-msft.  https://apiview.dev/Assemblies/Review/0b6337907c854077a23328f405a9cfdb/4c36ae42fb1b48b4b167881f2b25501e?diffRevisionId=3b6642b96574420da9012a005a4b5382&doc=False&diffOnly=True is a good place to discuss this.

Reply:
I was only referring to what looked like a bad merge but seems to be fixed now. As for the API itself yes you can leave that up to the ArchBoard review.